server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -810,16 +839,18 @@ def _delete_model_endpoint_monitoring_infra(self, uids: list[str], project: str)
             ModelMonitoringDriftMeasuresFile(project=project, endpoint_id=uid).delete()
             ModelMonitoringSchedulesFile(project=project, endpoint_id=uid).delete()
 
-        # delete tsdb records - NOT IMPLEMENTED
+        # delete tsdb records
         try:
-            # todo : delete tsdb records/tables for the model endpoint
-            # tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
-            #     project=project,
-            #     secret_provider=services.api.crud.secrets.get_project_secret_provider(
-            #         project=project
-            #     ),
-            # )
-            logger.info("TSDB resources were not deleted")
+            tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
+                project=project,
+                secret_provider=services.api.crud.secrets.get_project_secret_provider(
+                    project=project
+                ),
+            )
+            for uid in uids:
+                # todo: optimize to delete all in one call
+                tsdb_connector.delete_tsdb_records(endpoint_id=uid)
```

Comment:
⬆️

Reply:
@Eyal-Danieli is this deletion part of project deletion? regardless, worth putting effort into batch deletion (delete many uids at once / chunks) to ensure timeout wont expired per its operation

Reply:
it's not part of the project deletion process, and we don’t expect it to be called with a large number of model endpoints (like in the project deletion API). 
Usually, we expect it to be used when overwriting an existing model endpoint (which requires deleting the previous one) or when exceeding the archive limit for the same model endpoint and then you need to delete older versions.

Reply:
btw in project deletion process we delete tables and not specific records

Reply:
@liranbg @davesh0812 updated to batch deletion. Please note the differences between `v3io` and `tdengine`:
1. `v3io`: due to filter limitations, we delete data in chunks, with a max of 200 meps per chunk.
2. `tdengine`: Since each mep is stored in its own table and we cannot delete multiple tables in a single query, we continue to drop these tables individually. However, querying the table names is now done in a single query. We retrieve all relevant table names and then proceed to delete each table one by one.